resources/api-gateway/files/crd-apirules.yaml

```diff
@@ -433,7 +433,7 @@ spec:
                     type: array
                   path:
                     description: Path to be exposed
-                    pattern: ^/([0-9a-zA-Z./*]+)
+                    pattern: ^([0-9a-zA-Z./*(]+)
```

Comment:
I know it wasn't there before, but I would add a dash and underscore.
It is used here and there, for example: Some httpbin endpoints contain a dash (`response-headers`)

Reply:
You are right. I will add it.